deploy/role.yaml

```diff
@@ -1168,3 +1169,20 @@ rules:
   - get
   - watch
   - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-compliance
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - endpoints
+      - pods
```

Comment:
Why does prometheus need to list pods (I understand services and endpoints)?

Reply:
not sure exactly, I will investigate

Reply:
it's not that big of a deal since it's read-only -- I was just wondering